src/main/java/co/omise/models/Account.java

```diff
@@ -15,4 +19,24 @@ public String getEmail() {
     public void setEmail(String email) {
         this.email = email;
     }
+
+    /**
+     * public accessor method for the a {@link RequestBuilder<Account>} for getting the user's Account.
+     *
+     * @return the get request builder
+     */
+    public static GetRequestBuilder buildGetRequest() {
```

Comment:
I think we should move `GetRequestBuilder` to an inner class of `Account` to look more better builder pattern. 🤔 

```java
Request<Account> request = Account.GetRequestBuilder()
                                  .build();
```

Reply:
Will make changes according to the slack conversation around this topic.